src/components/layouts/FilterBar.tsx

```diff
@@ -210,7 +210,7 @@ export const FilterBar = ({
           direction="row"
           justifyContent="space-between"
           alignItems="center"
-          sx={{ display: { sm: 'flex', sd: 'none' }, mb: { xs: '12px', md: '0px' } }}
+          sx={{ display: { sm: 'flex', sd: 'none' }, mb: { xs: '12px', md: '0px' }, minWidth: '340px' }}
```

Comment:
This minWidth is giving unexpected issues. See below image - 

![Screenshot from 2024-07-09 14-18-29](https://github.com/copilot-platforms/tasks-app/assets/38468429/e026e2a0-d987-4c60-b7da-f7f3184f42bd)

The filter bar doesn't have side margins below 350px.

Reply:
Also, we are able to see border right on the top layout when <350px width.